.github/workflows/ci.yml

```diff
@@ -0,0 +1,17 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    container:
+      image: erlang:22.1
+    steps:
+      - uses: actions/checkout@v1
+      - run: rebar3 compile
```

Comment:
rebar3 runs the Gleam compiler as a before hook so we'll need to install that before somehow

Reply:
@lpil Ah yeah. Can I ask what the motivation was for separating the stdlib and compiler? Seems like it will be trickier to keep them in sync now as separate repos.

Reply:
rebar3 doesn't support subdirectories when using git deps so splitting it out into its own repo it means we can do more thorough testing by using the library in other applications and libraries

Reply:
@lpil What about injecting the standard lib as part of a custom compiler plugin? I'm not super familiar with Rebar, but I was looking into plugins for gleam-lang/gleam#300 and it seems like this would be relatively easy.

Reply:
I don't want to inject the standard library (as Elixir does) as this makes it harder to use from other languages and with other build tools. The stdlib is a normal Gleam library and as such can be best distributed using the normal package manager

Reply:
What about a GitHub action that automatically updated a `stdlib` branch on pushes to `master`? (e.g., https://github.com/marketplace/actions/push-git-subdirectory-as-branch)

This would allow the standard library to be imported as a git dependency while simplifying the process of updating it in lockstep with compiler changes.

Reply:
I'm not currently have any problems from two repos so until it becomes unwieldy I think it's best to continue as is without any extra tooling.

Reply:
Ok. My only argument would be that issues like gleam-lang/stdlib#2 could be caught immediately using a single repo setup.

Reply:
It would also have been caught by CI running on this repository so I don't see the advantage of merging them.

Reply:
Well the question is how should CI on this repo be structured in order to surface those kinds of issues?

If this repo targets the compiler’s master branch then it will become broken as changes are made to the compiler. Outstanding PRs will become unmergeable due to unrelated issues. 

We could potentially trigger CI builds on this repo based on merges to master in the compiler repo, but then where should those results be surfaced?

Reply:
This repo will use versions of Gleam that it supports. :)

Reply:
Doesn’t this repo have to target the compiler’s master branch on its master branch?

Reply:
Master is unstable so I shall be targeting releases instead

Reply:
Oh ok. I can update this workflow to target the latest release then.

That seems like it will make testing the master compiler more difficult since it won’t necessarily have a working standard library, and fixes to this library will have to wait until a compiler release is ready.

Reply:
I've been working with a branch for changes to the stdlib, and a release candidate compiler version can be used for testing major upgrades to libraries such as this one.